addon/form-object-cp.js

```diff
@@ -1,10 +1,15 @@
 import Ember from 'ember';
 import FormObject from './form-object';
 
-export default function formObject(model, impl = {}) {
-  return Ember.computed(model, function() {
-    return FormObject.extend(impl).create({
-      content: this.get(model)
+export default function formObject(modelProperty, Validations = {}, impl = {}) {
```

Comment:
for "implementation" - additional methods, properties etc. that are defined on the created buffer.